templates/documents/general_upload.html

```diff
@@ -33,6 +37,14 @@
 <input type="hidden" name="category_id" value="{$category_id}" />
 <input type="hidden" name="process" value="{$PROCESS}" />
 </form>
+<!-- Drag and drop uploader -->
+
+{php} if ($GLOBALS['document_drag_drop']) { {/php}
+<form method="post" enctype="multipart/form-data" action="{$WEBROOT}/interface/documents/upload.php?patient_id={php}echo $pid{/php}&parent_id={php}echo $category{/php}" class="dropzone">
```

Comment:
The general_list.php and the general_upload.php are shown concurrently on the same page.  The list loads the upload template file. So I had to put the js and css file in the list.php template file but the form action calls the /interface/document/upload.php with the necessary information pass to the document.php class file that completes the upload process.